src/program/services/downloaders/alldebrid.py

```
@@ -26,7 +26,7 @@ class AllDebridFile(BaseModel):
 
     n: str  # Name
     s: int  # Size in bytes
-    l: str  # Download link
+    l: str = ""  # Download link
```

Ruff (0.15.18)

[error] Ambiguous variable name: `l`

(E741)